containers/java/clipper-java-container/src/test/java/com/clipper/container/app/ContainerTest.java

```diff
@@ -0,0 +1,31 @@
+package com.clipper.container.app;
```

Comment:
Done. The main function of the RunContainer class takes five arguments: `<model_name>, <model_version>, <clipper_address>, <clipper_port>, <input_type>`.